shared/common-adapters/usernames/remote-container.js

```diff
@@ -3,31 +3,41 @@ import * as ProfileGen from '../../actions/profile-gen'
 import {Usernames} from '.'
 import {remoteConnect, compose} from '../../util/container'
 import * as Container from './container'
-import * as TrackerTypes from '../../constants/types/tracker'
+import * as UsersTypes from '../../constants/types/users'
 
 type OwnProps = Container.ConnectedProps
 type State = {|
-  broken: {[key: string]: TrackerTypes.TrackerState},
   following: Set<string>,
+  userInfo: {[key: string]: UsersTypes._UserInfo},
   username: string,
 |}
 
 // Connected username component
 const mapStateToProps = props => ({
-  _broken: props.broken,
   _following: props.following,
+  _userInfo: props.userInfo,
   _you: props.username,
 })
 
 const mapDispatchToProps = dispatch => ({
   _onUsernameClicked: (username: string) => dispatch(ProfileGen.createShowUserProfile({username})),
 })
 
+const userDatafromState = (stateProps, usernames) =>
+  usernames.map(username => ({
+    // $FlowIssue optional chain is needed
+    broken: stateProps._userInfo[username]?.broken || false,
```

Comment:
short term likely better to use the longer syntax and get rid this cause it'll mask real issues